server/src/main/java/org/opensearch/telemetry/tracing/TracerSettings.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing;
+
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.unit.TimeValue;
+
+/**
+ * Wrapper class to encapsulate tracing related settings
+ */
+public class TracerSettings {
```

Comment:
```suggestion
public class TelemetrySettings 
```

The settings should cover tracing and metrics, at least

Reply:
Done